### PR TITLE
[TASK] Prepare release notes for 11.1

### PR DESCRIPTION
**Pull request by dkd-friedrich**
# What this pr does

Prepares the release notes for EXT:solr 11.1, just an initial version
that has to be completed.

# How to test

Check the prepared release notes for 11.1

### CODE REVIEW

File: Documentation/Releases/solr-release-11-1.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+.. _releases-11-1:
+
+============================
+Apache Solr for TYPO3 11.1.0
+============================
+
+We are happy to release EXT:solr 11.1.0.
+The focus of this release has been on URL and SEO optimizations.
```
Comment (dkd-kaehm) [inline Documentation/Releases/solr-release-11-1.rst:16]: solarium upgrade to

File: Documentation/Releases/solr-release-11-1.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+There are many ways to get involved with Apache Solr for TYPO3:
+
+* Submit bug reports and feature requests on [GitHub](https://github.com/TYPO3-Solr/ext-solr)
+* Ask or help or answer questions in our [Slack channel](https://typo3.slack.com/messages/ext-solr/)
+* Provide patches through Pull Request or review and comment on existing [Pull Requests](https://github.com/TYPO3-Solr/ext-solr/pulls)
+* Go to [www.typo3-solr.com](http://www.typo3-solr.com) or call [dkd](http://www.dkd.de) to sponsor the ongoing development of Apache Solr for TYPO3
+
+Support us by becoming an EB partner:
+
+http://www.typo3-solr.com/en/contact/
```
Comment (dkd-kaehm) [inline Documentation/Releases/solr-release-11-1.rst:61]: https://shop.dkd.de/Produkte/Apache-Solr-fuer-TYPO3/

File: Documentation/Releases/solr-release-11-1.rst (hunk 1 above)
Comment (dkd-friedrich) [reply Documentation/Releases/solr-release-11-1.rst:16]: Added the Solarium upgrade and it's breaking changes to the "New in this release" section

File: Documentation/Releases/solr-release-11-1.rst (hunk 2 above)
Comment (dkd-friedrich) [reply Documentation/Releases/solr-release-11-1.rst]: Link updated

**Review by dkd-kaehm — APPROVED** [review]
can be merged if no tasks planned for that PR.

**Pull request merged**